packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md

```diff
@@ -145,6 +147,34 @@ customElements.define('my-element', MyElement);
 {% endswitchable-sample %}
 </pre>
 
+### Asides
+
+You can also insert an aside in your instructions by using the following format:
+
+```html
+<aside class="aside aside-positive">
```

Comment:
Nice use of `<aside>`. Would support converting all our existing `<p class="alert">` to this too.

Reply:
I went on google codelabs and looked at what element they used haha